Server/JokesOnYou.Web.Api/Models/BaseOwnedEntity.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace JokesOnYou.Web.Api.Models
+{
+    public abstract class BaseOwnedEntity
+    {
+        public int Id { get; set; }
+        public DateTime Created { get; set; }
```

Comment:
Created rather sounds like a bool type, perhaps we can rename it to CreationTime or anything else.

Reply:
removed this model.